pkg/iac/scanners/helm/parser/parser.go

```diff
@@ -76,6 +78,10 @@ func (p *Parser) SetAPIVersions(values ...string) {
 	p.apiVersions = values
 }
 
+func (p *Parser) SetKubeVersion(value string) {
+	p.kubeVersion, _ = chartutil.ParseKubeVersion(value)
```

Comment:
I think we should move the version parsing to the constructor and log an error if the user enters the wrong version.